bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,37 +1,39 @@
 #SearchBall
-@ChangeAction + action:searching, @LookAtFieldFeatures, @StandAndWait + duration:4, @TurnAround
+@ChangeAction + action:searching, @LookAtFieldFeatures, @WalkInPlace + duration:4, @TurnAround
 
 #DoNothing
-@ChangeAction + action:waiting, @LookForward, @StandAndWait
+@ChangeAction + action:waiting, @LookForward, @Stand
 
 #StandAndLook
-@ChangeAction + action:waiting, @LookAtFieldFeatures, @StandAndWait
+@ChangeAction + action:waiting, @LookAtFieldFeatures, @WalkInPlace
 
 #GetWalkreadyAndLocalize
-@ChangeAction + action:waiting + r:false, @LookForward + r:false, @PlayAnimationInit + r:false, @GetWalkready + r:false, @LookAtFieldFeatures + r:false, @StandAndWait + duration:2 + r:false, @StandAndWait
+@ChangeAction + action:waiting + r:false, @LookForward + r:false, @PlayAnimationInit + r:false, @GetWalkready + r:false, @LookAtFieldFeatures + r:false, @WalkInPlace
```

Comment:
```suggestion
@ChangeAction + action:waiting + r:false, @PlayAnimationInit + r:false, @LookAtFieldFeatures + r:false, @GetWalkready + r:false, @WalkInPlace
```
Since walkready is now used for the localization initialization, we should look at field features before.
I also removed LookForward, I think it is not needed anymore.

Reply:
but we dont move head during walkready, I think

Reply:
But then, we have to add the two seconds back because we need them for the localization. Are you sure that we cannot move the head during walkready? Does dynup set head positions during walkready?

Reply:
It looks like you're right. In dynup, isHeadZero is not set for `walkready` but still, the position defaults to `0` instead of just not publishing the joint positions for the head.

Reply:
I will change it so that we can move our head during walkready then this change will make sense